var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -225,6 +225,12 @@ def cmake_args(self):
         if "+sensei" in spec:
             args.append(self.define("SENSEI_DIR", spec["sensei"].prefix.lib.cmake))
 
+        # WarpX uses CCache by default, interfering with Spack wrappers
+        if spec.satisfies("@:24.01"):
+            args.append("-DCCACHE_PROGRAM=NO")
+        else:
+            args.append("-DWarpX_CCACHE=OFF")
```

Comment:
Nitpick, this matches style above more. Either works, so feel free to ignore or hit commit while I enable automerge...

```suggestion
        ccache_var = "CCACHE_PROGRAM" if spec.satisfies("@:24.01") else "WarpX_CCACHE"
        args.append(self.define(ccache_var, False))
```